dbms/src/Storages/Transaction/LearnerRead.cpp

```diff
@@ -172,7 +172,7 @@ LearnerReadSnapshot doLearnerRead(
     const auto & regions_info = mvcc_query_info.getRegionsInfo();
 
     // adjust concurrency by num of regions or num of streams * mvcc_query_info.concurrent
-    size_t concurrent_num = std::max(1, std::min(static_cast<size_t>(num_streams * mvcc_query_info->concurrent), regions_info.size()));
+    size_t concurrent_num = std::max(1, regions_info.size());
```

Comment:
May affect the performance？

Reply:
actually, the concurrent_num is always 1 by the following line `tmt.replicaReadMaxThread()`

https://github.com/pingcap/tiflash/blob/d09f3691f42569f7714327b5bf38afb7a9b8e264/dbms/src/Storages/Transaction/LearnerRead.cpp#L174-L178